stock_quant_package_package_type/models/product_product.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import models, tools
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _find_best_packaging(self, quantity):
+        self.ensure_one()
+        packagings = self.env["product.packaging"].search(
+            [("product_id", "=", self.id)], order="qty DESC"
```

Comment:
Just because we can have multiple packaging for the same qty
```suggestion
            [("product_id", "=", self.id)], order="qty DESC, sequence DESC"
```